docs/workbench-imagestreams.md

```
@@ -36,19 +36,19 @@ spec:
+### **Available annotations**
   - **`opendatahub.io/notebook-software:`** - a string that represents the technology stack included within the workbench image. Each technology in the list is described by its name and the version used (e.g. `'[{"name":"CUDA","version":"11.8"},{"name":"Python","version":"v3.9"}]`')
   - **`opendatahub.io/notebook-python-dependencies:`** -  a string that represents the list of Python libraries included within the workbench image. Each library is described by its name and currently used version (e.g. `'[{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"}]'`)
   - **`openshift.io/imported-from:`** - a reference to the image repository where the workbench image was obtained (e.g. `quay.io/repository/opendatahub/workbench-images`)
   - **`opendatahub.io/workbench-image-recommended:`** - a flag that allows the ImageStream tag to be marked as Recommended (used by the UI to distinguish which tags are recommended for use, e.g., when the workbench image offers multiple tags to choose from)
   - **`opendatahub.io/image-tag-outdated:`** - a flag to mark the ImageStream tag as outdated and out of regular maintenance cycle. The image tag will eventually be removed.
   - **`opendatahub.io/notebook-build-commit:`** - a reference to the build commit with the ID to look at the updated information.
 
-### **ImageStream definitions for the supported out-of-the-box images in ODH**  
+### **ImageStream definitions for the supported out-of-the-box images in ODH**
 
 The ImageStream definitions of the out-of-the-box workbench images for ODH can be found [here](https://github.com/opendatahub-io/notebooks/tree/main/manifests).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)